configuration/editors.md

```diff
@@ -0,0 +1,210 @@
+More details about the syntax of the different configuration files can be found in the [Configuration]({{base}}/configuration/index.html) chapter.
+
+Hint: Press `<Ctrl + N>` to create a new file.
+
+## Others
+
+### mcedit
+
+mcedit is an editor which comes with mc (Midnight Commander).
+
+
+#### Installing the syntax-files
+
+- download copy the syntax-files to */usr/share/mc/syntax/*
+- <https://groups.google.com/forum/#!searchin/openhab/syntax$20highlighting%7Csort:relevance/openhab/zJrfTDjahbc/r8GcpKDgr1cJ>
+
+- insert the following lines to the file *Syntax* in */usr/share/mc/syntax/*
+```shell
+    file ..\*\\.(items)$ openHAB\sItems 
+    include openhab-items.syntax  
+     
+    file ..\*\\.(sitemap)$ openHAB\sSitemap 
+    include openhab-sitemap.syntax
+     
+    file ..\*\\.(persist)$ openHAB\sPersistence
+    include openhab-persist.syntax
+     
+    file ..\*\\.(rules)$ openHAB\sRules
+    include openhab-rules.syntax 
+```
+- edit the Debian-line from
+`file (rules|rocks)$ Debian\srules`
+to 
+`file (rocks)$ Debian\srules`
+because it interferes with openHABs rules-files.
+
+### Notepad++
+
+Notepad++ Version 6.2 or above is required to support UDL2 (User Defined Language v2).
+
+#### Comments
+
+```shell
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"# Comments
+color brightgreen "//.*"
+color brightgreen start="/\*" end="\*/"
+color brightgreen start="/\*\*" end="\*/"
+```
+http://notepad-plus-plus.org/news/notepad-6.2-release-udl2.html
+
+#### How to import UDL2-files?
+
+- Download the UDL2-Files (openHAB-`*`.xml) from [openhab-samples](https://github.com/thefrip/openhab-samples)
+- Install the files like it is described in the [editors documentation](http://docs.notepad-plus-plus.org/index.php/User_Defined_Language_Files#How_to_install_user_defined_language_files)
+
+### vi
```

Comment:
Good question and i cant answer it. :D
The initial content is a 100% Fork from https://github.com/openhab/openhab1-addons/wiki/Syntax-Highlighting-for-external-editors

But the download repos readme is mentioning vim only,
so for me it would be ok to change this to "vim"